test/server/tests.js

```diff
@@ -115,7 +115,7 @@ describe("Basic", () => {
             .catch(err => done(err));
     });
 
-    it("should edit a new data source", done => {
+    it("should edit a new data source 2", done => {
```

Comment:
2? Could it not have a better description?

Reply:
I prefer not to comment, tests require a bit of reengineering.